server/src/test/java/org/elasticsearch/action/admin/indices/mapping/get/GetFieldMappingsResponseTests.java

```diff
@@ -49,4 +51,47 @@ public void testSerialization() throws IOException {
             }
         }
     }
+
+    @Override
+    protected GetFieldMappingsResponse doParseInstance(XContentParser parser) throws IOException {
+        return GetFieldMappingsResponse.fromXContent(parser);
+    }
+
+    @Override
+    protected GetFieldMappingsResponse createBlankInstance() {
+        return new GetFieldMappingsResponse();
+    }
+
+    @Override
+    protected GetFieldMappingsResponse createTestInstance() {
+        return new GetFieldMappingsResponse(randomMapping());
+    }
+
+    @Override
+    protected Predicate<String> getRandomFieldsExcludeFilter() {
+        return s -> true;
```

Comment:
what is the value of supporting unknown fields if the exclude filter always returns true?

Reply:
aren't there parts of the response where we could inject random fields?

Reply:
in fact not so much - but there are couple, fixed